backend/functions-db/posts.js

```diff
@@ -8,7 +8,7 @@ exports.createPost = (image, title, description, tags, userName, userId, callbac
     }
 
     // upload image somehow
-    var url = (image instanceof String) ? image : "testStringPlsFix";
+    var url = (typeof(image) === "string") ? image : "testStringPlsFix";
```

Comment:
use `const` instead of `var` and remove redundant parentheses around conditional